### PR TITLE
[improve](udf) support handle const column in java-udf

### PR DESCRIPTION
**Pull request by zhangstar333**
### What problem does this PR solve?

Problem Summary:
support handle const column in java-udf
the before and after execute time
```

mysql [ssb]>select count(test_query_qa.java_udf_test3(c_custkey,"asd")) from test_udf;
+-----------------------------------------+
| count(java_udf_test3(c_custkey, 'asd')) |
+-----------------------------------------+
|                                36000000 |
+-----------------------------------------+
1 row in set (0.68 sec)

------------

mysql [ssb]>select count(test_query_qa.java_udf_test3(c_custkey,"asd")) from test_udf;
+-----------------------------------------+
| count(java_udf_test3(c_custkey, 'asd')) |
+-----------------------------------------+
|                                36000000 |
+-----------------------------------------+
1 row in set (0.52 sec)


```

### Release note

None

### Check List (For Author)

- Test 
    - [ ] Regression test
    - [ ] Unit Test
    - [ ] Manual test (add detailed scripts or steps below)
    - [ ] No need to test or manual test. Explain why:
        - [ ] This is a refactor/code format and no logic has been changed.
        - [ ] Previous test can cover this change.
        - [ ] No code files have been changed.
        - [ ] Other reason 

- Behavior changed:
    - [ ] No.
    - [ ] Yes. 

- Does this need documentation?
    - [ ] No.
    - [ ] Yes. 

### Check List (For Reviewer who merge this PR)

- [ ] Confirm the release note
- [ ] Confirm test cases
- [ ] Confirm document
- [ ] Add branch pick label

### CODE REVIEW

**Comment by zhangstar333** [comment]
run buildall

**Comment by zhangstar333** [comment]
run buildall

**Comment by zhangstar333** [comment]
run buildall

File: be/src/vec/exec/jni_connector.cpp [hunk 1]
```diff
@@ -602,17 +602,28 @@ std::string JniConnector::get_jni_type(const TypeDescriptor& desc) {
 Status JniConnector::_fill_column_meta(ColumnPtr& doris_column, DataTypePtr& data_type,
                                        std::vector<long>& meta_data) {
     TypeIndex logical_type = remove_nullable(data_type)->get_type_id();
+    ColumnPtr const_doris_column = nullptr;
```
Comment (HappenLee) [inline be/src/vec/exec/jni_connector.cpp:605]: rename, should not call `const_doris_column`

**Review by HappenLee — APPROVED** [review]
LGTM

**Comment by zhangstar333** [comment]
run buildall

**Comment by zhangstar333** [comment]
run buildall

**Review by Mryange — APPROVED** [review]

**Pull request merged**